ghmirror/core/mirror_requests.py

```diff
@@ -83,10 +116,28 @@ def online_request(method, url, auth, data=None):
         if last_mod is not None:
             headers['If-Modified-Since'] = last_mod
 
-    resp = requests.request(method=method, url=url, headers=headers,
-                            timeout=REQUESTS_TIMEOUT)
+    resp = requests.request(method=method,
+                            url=url,
+                            headers=headers,
+                            timeout=REQUESTS_TIMEOUT,
+                            params=parameters)
```

Comment:
now some params are coming in the `url`, some more are injected with the `params`. A bit hard to keep track.. should w make `url` param-less and only use the `params` arg?

Reply:
Not sure how we want to do it. We can strip the URL removing the parameters that come from the client and then add the parameters in the `parameters` dict that's used in the github-mirror request.

Reply:
Yeah, I'd not reconstruct the URL in the flask app anymore... Just pass it as param-less url and always use the params argument when we do requests to GitHub. But only if that's not too convoluted.

Reply:
Up to you, just a suggestion

Reply:
I didn't saw that we were rebuilding the URL, make sense and leave all the parameters in only one place